blivet/formats/__init__.py

```diff
@@ -367,6 +367,15 @@ def create(self, **kwargs):
         """
         log_method_call(self, device=self.device,
                         type=self.type, status=self.status)
+        self._preCreate(**kwargs)
+        try:
+            self._create(**kwargs)
+        except Exception:
+            raise
```

Comment:
That is there because a later commit will be adding code to the except clause, but it's a good idea to add the try/except later since the additional complexity achieves nothing at this point.